src/collections/config/types/vectorIndex.ts

```diff
@@ -9,7 +9,7 @@ export type VectorIndexConfigHNSW = {
   filterStrategy: VectorIndexFilterStrategy;
   flatSearchCutoff: number;
   maxConnections: number;
-  quantizer: PQConfig | BQConfig | SQConfig | undefined;
+  quantizer: PQConfig | BQConfig | SQConfig | RQConfig | undefined;
```

Comment:
Looking at this with fresh eyes, should we use `QuantizerConfig` here to reduce the duplication? So: `quantizer: QuantizerConfig | undefined`?

Reply:
good idea 👍